src/payout/repo.rs

```diff
@@ -16,90 +34,45 @@ impl Payouts {
         Self { pool: pool.clone() }
     }
 
-    #[instrument(name = "payouts.create", skip(self, tx))]
-    pub async fn create_in_tx(
-        &self,
-        tx: &mut Transaction<'_, Postgres>,
-        new_payout: NewPayout,
-    ) -> Result<PayoutId, PayoutError> {
-        sqlx::query!(
-            r#"INSERT INTO bria_payouts (id, account_id, wallet_id, payout_queue_id, profile_id, external_id)
-               VALUES ($1, $2, $3, $4, $5, $6)"#,
-            new_payout.id as PayoutId,
-            new_payout.account_id as AccountId,
-            new_payout.wallet_id as WalletId,
-            new_payout.payout_queue_id as PayoutQueueId,
-            new_payout.profile_id as ProfileId,
-            new_payout.external_id,
-        ).execute(&mut **tx).await?;
-        let id = new_payout.id;
-        EntityEvents::<PayoutEvent>::persist(
-            "bria_payout_events",
-            tx,
-            new_payout.initial_events().new_serialized_events(id),
-        )
-        .await?;
-        Ok(id)
-    }
-
-    #[instrument(name = "payouts.find_by_id", skip(self))]
-    pub async fn find_by_id(
+    pub async fn find_by_account_id_and_id(
         &self,
         account_id: AccountId,
         payout_id: PayoutId,
     ) -> Result<Payout, PayoutError> {
-        let rows = sqlx::query!(
+        let payout = es_entity::es_query!(
+            "bria",
+            &self.pool,
             r#"
-          SELECT b.*, e.sequence, e.event
-          FROM bria_payouts b
-          JOIN bria_payout_events e ON b.id = e.id
-          WHERE account_id = $1 AND b.id = $2
-          ORDER BY b.created_at, b.id, e.sequence"#,
+            SELECT *
+            FROM bria_payouts
+            WHERE account_id = $1 AND id = $2"#,
             account_id as AccountId,
```

Comment:
why we are using es_query!() in other places we first get the entity and then check if account id matches ?